api/v1_projects.inc

```diff
@@ -866,7 +866,8 @@ function api_v1_project_page(string $method, array $data, array $query_params)
             case 'revert':
                 return $proof_project_page->save_and_revert(receive_project_text_from_request_body('text'));
```

Comment:
(I spotted my earlier mistake here with the text parameter. I'll make a PR to fix it after this so it doesn't conflict.)